docs/man/docker-create.1.md

```diff
@@ -74,6 +75,13 @@ IMAGE [COMMAND] [ARG...]
 **--cpuset-cpus**=""
    CPUs in which to allow execution (0-3, 0,1)
 
+**--cpuset-mems**=""
+   Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems.
+
+   If you have four memory nodes on your system(0-3), use `--cpuset-mems=0,1`
+then processes in your Docker container will only use memory from the first
+two memory nodes.
```

Comment:
This sentence doesn't "run" properly. Thinking something like (but I'm no doc-writer);

On a system with four memory nodes (0 - 3), setting `--cpusets-mems=0,1` will
restrict the Docker container to only use memory from the first two memory nodes.